.github/workflows/publish-docker-image.yml

```diff
@@ -0,0 +1,34 @@
+name: publish-docker-image
+
+on:
+  push:
+    branches: [ main ]
+
+env:
+  DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+
+jobs:
+  docker:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Java 19
+        uses: actions/setup-java@v3
+        with:
+          distribution: corretto
+          java-version: 19
+      
+      - name: Login to Docker
+        run: docker login -u $DOCKER_USERNAME -p $DOCKER_HUB
+
+      - name: Get Commit Time
+        run: echo "COMMIT_TIME=$(git show -s --format=%ci ${{ github.sha }})" >> $GITHUB_ENV`s
+
+      - name: Build and push Docker image
+        run: gradle jib --image="$DOCKER_USERNAME/file-backup:${{ env.COMMIT_TIME }}"
```

Comment:
Actually there are 2 images `file-backup-local` and `file-backup-s3`.

They have config in their respective `build.gradle` file

Reply:
Made the repos https://hub.docker.com/repositories/wilmol